homeassistant/components/switcher_kis/light.py

```diff
@@ -39,12 +39,29 @@ def async_add_light(coordinator: SwitcherDataUpdateCoordinator) -> None:
             coordinator.data.device_type.category
             == DeviceCategory.SINGLE_SHUTTER_DUAL_LIGHT
         ):
-            entities.extend(SwitcherDualLightEntity(coordinator, i) for i in range(2))
+            entities.extend(SwitcherMultiLightEntity(coordinator, i) for i in range(2))
         if (
             coordinator.data.device_type.category
             == DeviceCategory.DUAL_SHUTTER_SINGLE_LIGHT
         ):
             entities.append(SwitcherSingleLightEntity(coordinator, 0))
+        if coordinator.data.device_type.category == DeviceCategory.LIGHT:
```

Comment:
This guard has no effect since you check all the individual models so it can be removed (only check the `DeviceType`)